content/en/docs/concepts/security/hardening-guide/scheduler.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Hardening Guide - Scheduler Configuration"
+description: >
+    Information about how to make the Kubernetes scheduler more secure.
+content_type: concept
+draft: false
+weight: 90
+---
+
+<!-- overview -->
+[kube-scheduler](/docs/reference/command-line-tools-reference/kube-scheduler) is one of the critical components of the control plane. Generally, it the of the top consideration for the security of the Control Plane. 
+
+In this document we explain the effects of a misconfigured scheduler and how we can make the scheduler more secure.
+
+A misconfigured Scheduler can have security implications. Such a scheduler can target specific nodes and evict the services sharing the node. This can aid an attacker with a [Yo-Yo attack](https://arxiv.org/abs/2105.00542) which is an attack on a vulnerable autoscaler.
+
+<!-- body -->
+## kube-scheduler configurations
+
+### Scheduler Authentication & Authorization configuration options
+{{<table caption="Authentication and Authorization Configurations">}}
+| Configuration | Description | Security Hardening Advice |
+| --- | :--- | --- |
+|`authentication-kubeconfig`|kubeconfig file pointing at the 'core' kubernetes server with enough rights to create tokenreviews.authentication.k8s.io. This is optional. If empty, all token requests are considered to be anonymous and no client CA is looked up in the cluster. |Make sure that a proper to provide a proper kubeconfig so that the server calls are secure. This kubeconfig file should also maintained securely.|
+|`authentication-tolerate-lookup-failure`| If true, failures to look up missing authentication configuration from the cluster are not considered fatal. Note that this can result in authentication that treats all requests as anonymous. |Set to `false` to make sure invalid authentication configurations do not lead to requests passing off as anonymous |
+|`authentication-skip-lookup`| If false, the authentication-kubeconfig will be used to lookup missing authentication configuration from the cluster. |This should be set to `false` to make sure all missing authentication configuration falls back to the authentication kubeconfig.|
```

Comment:
Consider switching this statement to start with "If true" to improve consistency with the rest of the document